lib/nightmare.js

```diff
@@ -91,6 +91,22 @@ function Nightmare(options) {
     debug('electron child process exited with code ' + code + ': ' + help[code]);
   });
 
+  Object.defineProperty(this, 'engineVersions', {
+    get: function() {
+      delete this.engineVersions; 
+      return this.engineVersions = JSON.parse(
```

Comment:
(Also you can’t have `writable: true` if you have a getter or setter. So you need to actually call `defineProperty` again or just keep the getter and not call `execSync`)